app/notifications/notifications_ses_callback.py

```diff
@@ -75,7 +77,7 @@ def process_ses_response(ses_request):
                 )
 
             create_outcome_notification_statistic_tasks(notification)
-
+            send_delivery_status_to_service.apply_async([notification.id], queue=QueueNames.NOTIFY)
```

Comment:
Need to wrap notification_id with str(notification_id) otherwise there will be a serialisation error

Reply:
changed in 3594fc.